### DISCUSSION TITLE
How to remove user?

### DISCUSSION THREAD

**Discussion by giiska** [verdaccio/verdaccio#3240] [Q&A]
I currently remove user by modifing htpasswd file which need restart docker container. I think this is not good, but can't find any document about this.

**Comment by juanpicado** [comment]
Nop possible with tooling, at this point is manual.

**Comment by edclement** [comment]
You can try using a different authentication plugin: https://verdaccio.org/docs/plugin-auth#list-community-authentication-plugins. Another option is to develop your own authentication plugin.
